bazel/external/dpdk.BUILD

```diff
@@ -0,0 +1,75 @@
+# bazel/external/dpdk.BUILD
+
+# Copyright 2020-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load("@//bazel/rules:package_rule.bzl", "pkg_tar_with_symlinks")
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "dpdk_sde",
+    srcs = glob([
+        "dpdk-bin/lib/libbf_switchd_lib.so*",
+        "dpdk-bin/lib/libdriver.so",
+        "dpdk-bin/lib/libtarget_sys.so",
```

Comment:
Do we need target_utils as well here?

Reply:
Similar question about dpdk library, earlier we linked with dpdk_infra

Reply:
These definitions are sufficient for `bazel build //stratum/hal/bin/tdi/dpdk:stratum_dpdk --define target=dpdk` to complete without error.